server/src/main/java/org/jboss/as/server/logging/ServerLogger.java

```diff
@@ -1354,23 +1365,10 @@
     @Message(id = Message.NONE, value = "The attribute '%s' has changed from '%s' to '%s'")
     String jmxAttributeChange(String name, String oldState, String stateString);
 
-    @LogMessage(level = INFO)
-    @Message(id = Message.NONE, value = "The configuration history is managed through Git")
-    void usingGit();
-
     @Message(id = Message.NONE, value = "Repository initialized")
```

Comment:
It's suspicious when I see a `@Message` without an ID. Most of the time, I consider that this should be a DEBUG log and in that case, I don't see which benefit we have to use a @Message for it instead of a plain `debugf()` call.

In that case, it turns out that this is a String used in a Git commit:

```
git.commit().setSign(gitConfig.isSign()).setMessage(ServerLogger.ROOT_LOGGER.repositoryInitialized()).call();
```

I'm not sure what we want to do here... 
Do we need to i18nize this commit message?
Should it be a stable message with a Message ID that can be tracked in a knowledge base?

Reply:
@darranl that's not directly related to your PR though :)

Reply:
IMO these should not have an ID - these messages are used for the commit messages in the git repository so similar to how we don't have message IDs for internationalised CLI messages we don't have message IDs for internationalised commit messages.